streams/src/main/java/org/apache/kafka/streams/kstream/Aggregator.java

```diff
@@ -17,22 +17,36 @@
 
 package org.apache.kafka.streams.kstream;
 
+import org.apache.kafka.common.annotation.InterfaceStability;
+
 /**
  * The {@link Aggregator} interface for aggregating values of the given key.
- *
- * @param <K>   key type
- * @param <V>   original value type
- * @param <T>   aggregate value type
+ * This is a generalization of {@link Reducer} and allows to have different types for input value and aggregation
+ * result.
+ * {@link Aggregator} is used in combination with {@link Initializer} that provides an initial aggregation value.
+ * <p>
+ * {@link Aggregator} can be used to implement aggregation functions like count.
+
+ * @param <K> key type
+ * @param <V> input value type
+ * @param <VA> aggregate value type
+ * @see Initializer
+ * @see KGroupedStream#aggregate(Initializer, Aggregator, org.apache.kafka.common.serialization.Serde, String)
+ * @see KGroupedStream#aggregate(Initializer, Aggregator, org.apache.kafka.streams.processor.StateStoreSupplier)
+ * @see KGroupedStream#aggregate(Initializer, Aggregator, Windows, org.apache.kafka.common.serialization.Serde, String)
+ * @see KGroupedStream#aggregate(Initializer, Aggregator, Windows, org.apache.kafka.streams.processor.StateStoreSupplier)
+ * @see Reducer
  */
-public interface Aggregator<K, V, T> {
+@InterfaceStability.Unstable
+public interface Aggregator<K, V, VA> {
```

Comment:
It's not about the number of letters. But what does `T` stand for? "Type" is not very meaningful. `VA` is for "Value Aggregate". (ie, `K` -> "key type" -- `V` -> "value type" -- "value aggregate type"). This is more meaningful -- in some other classes we also use stuff like `K`, `K1` -- this in no helpful at all to the user -- it almost like using `A`, `B`.